### PR TITLE
Implemented notification related functions

### PR DESCRIPTION
**Pull request by hazerb**
Implemented notification related functions and changed old ones if it was necessary.Also fixed minor things.Please review.

### CODE REVIEW

**Review by EnesToptas — APPROVED** [review]
Everything looks great, I have just commented about some coding habits, that you might find useful. We can merge this.

File: backend/controllers/notificationController.js [hunk 1]
```diff
@@ -1,42 +1,56 @@
...
-        isRead : req.body.isRead,
-        title : req.body.title,
-        body : req.body.body,
-        link : req.body.link,
-        other : req.body.other,
+const {User, Project, Notification}  = require('../model/db')
+
+deleteNotification = async function(req,res){
+    try{
+	id = req.params.id
```
[the remark this answers is not in the document]
Comment (hazerb) [reply backend/controllers/notificationController.js:5]: thank you for recommendation

File: backend/controllers/notificationController.js [hunk 2]
```diff
@@ -1,42 +1,56 @@
...
     }
 }
 
 
 
 module.exports = {
-    addNotification,
-    getNotifications,
+    deleteNotification,
+    getNotifications
```
[the remark this answers is not in the document]
Comment (hazerb) [reply backend/controllers/notificationController.js:55]: thank you for recommendation

File: backend/controllers/notificationController.js [hunk 3]
```diff
@@ -1,42 +1,56 @@
...
-        link : req.body.link,
-        other : req.body.other,
+const {User, Project, Notification}  = require('../model/db')
+
+deleteNotification = async function(req,res){
+    try{
+	id = req.params.id
+	await Notification.destroy({
+	    where : {
+	        id : id
```
[the remark this answers is not in the document]
Comment (hazerb) [reply backend/controllers/notificationController.js:8]: thank you for recommendation

**Comment by hazerb** [comment]
okay thank you im merging it

**Pull request merged**